Source/UI/Authentication/Scenes/EnterPassword/EnterPasswordViewController.swift

```diff
@@ -456,7 +456,10 @@ open class EnterPasswordViewController: LimeAuthUIBaseViewController, EnterPassw
         self.updateViews()
         actionFeedback?.scene(.operationSuccess)
         
+        // we're delaying auto-navigation, so disable any tempering with potentional modal presentation
+        setGestureDismissEnabled(to: false)
```

Comment:
I would also disable the gesture when there's a pending activity (see `executeOperation()` function). We don't want to handle cancelation of pending operation, when user dismisses the controller in the wrong time.

Reply:
OK, makes sense